tests-integration/tests/cosmwasm/helper.rs

```diff
@@ -13,9 +13,10 @@ pub fn dummy_msg_info() -> MessageInfo {
     message_info(&creator, &coins(1000, "ibc"))
 }
 
-pub fn dummy_checksum() -> Vec<u8> {
+pub fn dummy_checksum() -> Binary {
```

Comment:
can we use `Checksum` type alias here?

Reply:
I ended up using `cosmwasm_std::Checksum`
[8181b44](https://github.com/cosmos/ibc-rs/pull/1272/commits/8181b44a01d619d7a946f2390ae2b36fdec8c6d2)